core/primitives/src/version.rs

```diff
@@ -131,7 +131,9 @@ lazy_static! {
             STABLE_PROTOCOL_FEATURES_TO_VERSION_MAPPING.iter()
         {
             assert!(
-                PROTOCOL_FEATURES_TO_VERSION_MAPPING[&stable_protocol_feature]
+                *nightly_protocol_features_to_version_mapping
+                    .get(&stable_protocol_feature)
```

Comment:
```suggestion
                    .get(stable_protocol_feature)
```

nit: this deref is probably unnneded.